config.example

```diff
@@ -8,5 +8,6 @@
     "nickserv_password": "yourpasswd",
     "auxchans": [],
     "extendedcommands": ["jaden","excuse","dong"],
-    "topmodifier": 0.05
+    "topmodifier": 0.05,
+    "repolink": "https://github.com/tonyravioli/dongerdong"
```

Comment:
I can update to not use a config var if that's the preference but like I said, in my projects this is exactly something I would consider a config element. No biggie tho